extension.neon

```diff
@@ -8,6 +8,7 @@ parameters:
 		containerXmlPath: null
 		constantHassers: true
 		consoleApplicationLoader: null
+		invalidateResultCacheOnDiDefinitionChange: true
```

Comment:
I don't think we need an option for this.

Reply:
I thought it would be better to provide the toggle so people can opt-out if they find this feature somehow stepping in their way of doing things, while continuing to use Symfony extension. I can remove it if you think it's better to always have it enabled.

Reply:
Sure, leave it always enabled until we get feedback we can act on.

Reply:
I mean remove the option to disable the extension.

Reply:
Configurability was removed.